source/Octopus.Tentacle/Kubernetes/IKubernetesPodLogService.cs

```diff
@@ -57,7 +51,28 @@ public async Task<(IReadOnlyCollection<ProcessOutput> Outputs, long NextSequence
             var combinedLogs = podLogs.Outputs.Concat(tentacleLogs).OrderBy(o => o.Occurred).ToList();
             return (combinedLogs, podLogs.NextSequenceNumber);
 
-            async Task<(IReadOnlyCollection<ProcessOutput> Outputs, long NextSequenceNumber, int? ExitCode)> ReadPodLogs(Stream stream)
+            async Task<(IReadOnlyCollection<ProcessOutput> Outputs, long NextSequenceNumber, int? ExitCode)> GetPodLogs()
+            {
+                var sinceTime = scriptPodSinceTimeStore.GetSinceTime(scriptTicket);
+                try
+                {
+                    return await GetPodLogsWithSinceTime(sinceTime);
+                }
+                catch (UnexpectedPodLogLineNumberException ex)
+                {
+                    Log.Warn(ex, "Unexpected Pod log line numbers found, retrying with loading ");
```

Comment:
Should we try log this in the task log?

Reply:
yeah, I think that's a good idea

Reply:
I would say it's probably not worth putting in the task log, if it's a "real" issue, then it'll fail again when we retry and that error will bubble up. If it's just a little blip that we handle automatically it's not worth bothering the user about.

Reply:
I didn't see Al's comment before I submitted my review. Adding it to the task log if it's Verbose is probably fine, I'd avoid making it a warning or info level though. 👍